app/views/hyrax/base/_work_type.html.erb

```diff
@@ -0,0 +1,6 @@
+<h3 class="work-type-tag">
+  <% if Hyrax.config.display_work_icon? %>
+    <span class="<%= t("hyrax.icons.#{presenter.model}") %>" aria-hidden="true"></span>
```

Comment:
Please don't put configuration in translations. This requires one change per language we support. Add it to configuration.

Reply:
This is already in the translations (see: https://github.com/samvera/hyrax/blob/master/lib/generators/hyrax/work/templates/locale.en.yml.erb#L4)

Open to suggestion if I'm missing something here. :-)

Reply:
😢

Reply:
Should this be `Sufia::ModelIcon.css_class_for(presenter.model)`

Reply:
Assuming you mean `Hyrax::ModelIcon.css_class_for` right?

Reply:
That's the one.

Reply:
I think you are right here, but it brings up a couple of issues. The way things sit right now, I get an error that `model_name` doesn't doesn't exist for `GenericWork`. However, if I rename my `model` method to `model_name` I get a different error. I'm wondering if we should ticket that separately with more information, probably implementing a proper `model_name` method on works.